src/Styles.css

```diff
@@ -0,0 +1,213 @@
+@import url('https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap');
```

Comment:
Don't put all styles in one file. Break it down by components and put in separate files.